tests/test_hepnames.py

```diff
@@ -1052,6 +1084,56 @@ def test_advisors_from_701__a_g_i():
     assert expected == result['701']
 
 
+def test_advisors_from_701__a_g_i_h():
+    schema = load_schema('authors')
+    subschema = schema['properties']['advisors']
+
+    snippet = (
+        '<datafield tag="701" ind1=" " ind2=" ">'
+        '  <subfield code="a">Rivelles, Victor O.</subfield>'
+        '  <subfield code="g">PhD</subfield>'
+        '  <subfield code="i">INSPIRE-00120420</subfield>'
+        '  <subfield code="x">991627</subfield>'
+        '  <subfield code="y">1</subfield>'
+        '  <subfield code="h">HIDDEN</subfield>'
+        '</datafield>'
+    )  # record/1474091
```

Comment:
did you invent this data? if so you should say that in the comment (`synthetic data`).